pytg/argument_types.py

```diff
@@ -85,6 +85,12 @@ def parse(self, value):
 		return super(User, self).parse(value)
 
 
+class Channel(Peer):
```

Comment:
This is a duplicate of [line 100](https://github.com/luckydonald/pytg/pull/61/files#diff-015c52c157d46f032285dfebe38d0c7aR100)...?

Will fix this.